README.md

```diff
@@ -448,45 +361,49 @@ Our branches (in order of stability are):
 2.  Add your changes to the CHANGELOG.md file under `## [Unreleased]`. See [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for formatting docs.
 3.  Create a pull request of the `feature` branch _into_ `main`.
 4.  After a feature branch has approved accessibility review and VQA, merge `feature` branch into `main`, then merge `main` branch into `qa` for testing. Merging can be done manually or with a pull request. Pushing the changes to the remote branch will automatically trigger a deployment to the `qa` environment.
+
+    4a. If you introduced a new path in your PR, update the reverse proxy QA configuration so that it will be accessible in the `qa` environment.
+
 5.  After QA has tested and approved cut a `release` branch off of `qa` using our Release Naming Strategy. Please see Release Naming Strategy for details on how we plan to number our releases during the initial rollout of this project.
 
-`git checkout -b release/x.y.z`.
+    `git checkout -b release/x.y.z`.
+
+    (see tagged releases or changelog for last version)
 
-(see tagged releases or changelog for last version)
+6.  Update CHANGELOG.md in release branch by moving updates from unreleased into the new release section.
 
-6. Update CHANGELOG.md in release branch by moving updates from unreleased into the new release section.
+    ie. `## [2.4.13] - 04-21-2023`
 
-   ie. `## [2.4.13] - 04-21-2023`
+7.  Commit and push changes to release branch.
+8.  The `production` branch is protected. When the release is ready, create a pull request to merge `release` branch to `production`. Pushing to `production` will automatically deploy to the production environment.
+9.  Immediately after merging release to production, finish the release. This is done by backmerging the release to `qa` and `main` followed by creating the tags/release on github using these commands:
 
-7. Commit and push changes to release branch.
-8. The `production` branch is protected. When the release is ready, create a pull request to merge `release` branch to `production`. Pushing to `production` will automatically deploy to the production environment.
-9. Immediately after merging release to production, finish the release. This is done by backmerging the release to `qa` and `main` followed by creating the tags/release on github using these commands:
+    `$ git tag -a x.y.z`
+    `$ git push origin --tags`
 
-   `$ git tag -a x.y.z`
-   `$ git push origin --tags`
+    9a. If you introduced a new path in your PR, update the reverse proxy production configuration (merging the change you already made in 4a!) so it will be accessible in the `production` environment.
 
-### Release Naming Strategy
+### Release naming strategy
```

Comment:
I don't have a strong opinion but why the sentence case? UX patterns is to use sentence case for titles and I'm cool with this being a convention we follow on our docs.

Reply:
I just prefer it and the casing was inconsistent, so i wanted to pick one